### DISCUSSION TITLE
青龙面板无法在定时任务查看日志

### DISCUSSION THREAD

**Discussion by Ayers0** [shanmiteko/LotteryAutoScript#219] [Q&A] [answered]
点击查看日志报错 Cannot read properties of undefined (reading 'lastIndexOf')
在右侧任务日志的日志文件可以正常查看

青龙面板版本：v2.10.13

**Comment by shanmiteko** [comment, marked as answer]
来几张截图

**Reply by Ayers0** [reply]
![QQ图片20221106171527](https://user-images.githubusercontent.com/117542293/200162972-600cad4f-0654-43b0-b146-7f77b3f9ccd4.png)
![QQ图片20221106171631](https://user-images.githubusercontent.com/117542293/200163008-ca279d6b-1d83-42dc-a35a-ba5b41059b83.png)

**Reply by shanmiteko** [reply]
多半是没有设置对

**Reply by Ayers0** [reply]
感谢，问题已经解决了，在定时任务命令那少输了task
重装过青龙面板，没挂梯子GitHub的教程加载不出来，凭记忆添加的定时任务

**Reply by Ayers0** [reply]
再厚脸皮提个小建议，如果脚本能够直接读取青龙面板配置文件的Cookie那就更好了

**Comment by Houyiran-cool** [comment]
> 点击查看日志报错 Cannot read properties of undefined (reading 'lastIndexOf') 在右侧任务日志的日志文件可以正常查看
> 
> 青龙面板版本：v2.10.13

你现在还在抽奖吗 会识别验证码那个ocr嘛我想问问你
